.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
please remove non project files and add them into a gitignore file, so they will not be committed